pkg/utils/manager/manager.go

```diff
@@ -0,0 +1,60 @@
+package manager
+
+import (
+	"context"
+	"sync"
+	"time"
+)
+
+// Manager keeps track of the go routines in the system and can be used to gracefully shutdown the
+// the system by waiting for completion of go routines added to it.
+type Manager interface {
```

Comment:
Lets rename this to `type Interface interface` so import it would be `manager.Interface`

Reply:
done